libs/agora/api-description/src/openapi.yaml

```diff
@@ -18,3 +19,7 @@ tags:
 paths:
   /dataversion:
     $ref: paths/dataversion.yaml
+  /getTeams:
+    $ref: paths/teams.yaml
+  /team-member/{name}/image:
```

Comment:
The endpoint should be `/team-members/{name}/image`.

Reply:
updated

Reply:
kept kebab case as it seems to be preferred for SEO.  

Why Kebab Case is Preferred

    Readability: Hyphens make the URL structure clear and easy to understand [2](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/Why-you-should-make-kebab-case-a-URL-naming-convention-best-practice).

    Search Engine Optimization (SEO): Search engines treat hyphenated words as separate terms, improving SEO [1](https://ux.stackexchange.com/questions/41595/what-is-the-casing-convention-for-url-routes).

    Consistency: Using lowercase and hyphens creates a consistent and predictable URL structure [2](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/Why-you-should-make-kebab-case-a-URL-naming-convention-best-practice).

    Compatibility: Kebab case works well across different operating systems and servers, regardless of case sensitivity settings [1](https://ux.stackexchange.com/questions/41595/what-is-the-casing-convention-for-url-routes).

Reply:
The quote format ("> ") makes the text easier to read:

> Readability: Hyphens make the URL structure clear and easy to understand [2](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/Why-you-should-make-kebab-case-a-URL-naming-convention-best-practice).

> Search Engine Optimization (SEO): Search engines treat hyphenated words as separate terms, improving SEO [1](https://ux.stackexchange.com/questions/41595/what-is-the-casing-convention-for-url-routes).

> Consistency: Using lowercase and hyphens creates a consistent and predictable URL structure [2](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/Why-you-should-make-kebab-case-a-URL-naming-convention-best-practice).

> Compatibility: Kebab case works well across different operating systems and servers, regardless of case sensitivity settings [1](https://ux.stackexchange.com/questions/41595/what-is-the-casing-convention-for-url-routes).

Reply:
Using kebab case for paths make sense to me and it's one of the two conventions I've been considering changing for OpenChallenges (the other one is the naming convention for schema parameter). [GitHub OAS](https://raw.githubusercontent.com/github/rest-api-description/main/descriptions/ghes-3.9/ghes-3.9.yaml), which I've used as a reference for the OC OAS, uses kebab case for the path.

Most importantly, I would like us to agreeing on a convention that is then applied to all the projects in the monorepo.

We can then configure the OAS linter to apply this convention, ideally at the workspace level, and then no longer think about it until someone suggests to review this convention.

Here is the protocol I propose to review and validate the convention:

1. Invite all the developers who work in the monorepo to discuss this convention (at least one developer per product).
2. One or more people describe how they would change the convention and why
3. Hopefully there is a consensus and no strong opposition
4. The convention is applied and is ideally enforced by a linter

Would you agree to bring this case to an upcoming meeting?

Reply:
@sagely1 Feel free to use kebab-case for now and resolve this comment. After we settle on a format, we can use the IBM OpenAPI Validator rule https://github.com/IBM/openapi-validator/blob/main/docs/ibm-cloud-rules.md#ibm-path-segment-casing-convention to enforce the format (currently turned off because OC and other projects don't use the IBM convention).

Reply:
sounds good!